src/components/ChatLog.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+
```

Comment:
This would have been a great place to import the ChatLog.css file to style the ChatLog!